pull/github.go

```diff
@@ -576,6 +669,11 @@ func (c *v4Commit) ToCommit() *Commit {
 	}
 }
 
+type v4Collaborator struct {
```

Comment:
Let's reuse `v4Actor` in the places where this is used instead of defining a new type, since the fields are the same.